tests/integrations/protocol/library.cairo

```diff
@@ -1367,14 +1492,37 @@ namespace anyone_instance {
         return (address=address,);
     }
 
+    func offset{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() {
+        let (carbonable_offseter) = carbonable_offseter_instance.get_address();
+        let (caller) = get_address();
+        with carbonable_offseter {
+            let (success) = carbonable_offseter_instance.offset(caller=caller);
+            assert success = TRUE;
+        }
+        return ();
+    }
+
+    func snapshot{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() {
```

Comment:
`anyone.snapshot()` is never called, so we never test that `anyone` is not allowed to call `snapshot()`. 
Do we want to keep that way ?